Gemfile.lock

```diff
@@ -1704,30 +1820,29 @@ GEM
       simplecov_json_formatter (~> 0.1)
     simplecov-html (0.12.3)
     simplecov_json_formatter (0.1.4)
-    sinatra (3.1.0)
+    sinatra (4.0.0)
```

Comment:
this may hurt, better keep the 3.x for sinatra

Reply:
Locked down to 3.x.